services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/ApplicationConfig.java

```diff
@@ -317,6 +317,18 @@ public ServiceConfigProperties httpServerProperties() {
         return new ServiceConfigProperties();
     }
 
+    /**
+     * Gets properties for configuring an authentication mechanism for the HTTP based 
+     * Device Registry Management endpoint.
+     *
+     * @return The properties.
+     */
+    @Bean
+    @ConfigurationProperties(prefix = "hono.registry.http.auth")
+    public MongoDbBasedRegistryAuthenticationConfigProperties httpServerAuthConfigProperties() {
```

Comment:
wouldn't it make more sense to simply create an HTTP specific subclass of `ServiceConfigProperties` and add the required properties there?

Reply:
It makes more sense and I have now created a _subclass_ of `ServiceConfigProperties` to make the _realm_ configurable.